src/languages/es.js

```diff
@@ -1183,4 +1183,471 @@ export default {
+            stateISO: 'UT',
+            stateName: 'Utah',
+        },
+        VA: {
+            stateISO: 'VA',
+            stateName: 'Virginia',
+        },
+        VT: {
+            stateISO: 'VT',
+            stateName: 'Vermont',
+        },
+        WA: {
+            stateISO: 'WA',
+            stateName: 'Washington',
+        },
+        WI: {
+            stateISO: 'WI',
+            stateName: 'Wisconsin',
+        },
+        WV: {
+            stateISO: 'WV',
+            stateName: 'Virginia Occidental',
+        },
+        WY: {
+            stateISO: 'WY',
+            stateName: 'Wyoming',
+        },
+        DC: {
+            stateISO: 'DC',
+            stateName: 'District Of Columbia',
+        },
+    },
+    allCountries: {
+        AC: 'Isla Ascensión',
+        AD: 'Andorra',
+        AE: 'Emiratos Árabes Unidos',
+        AF: 'Afganistán',
+        AG: 'Antigua y Barbuda',
+        AI: 'Anguila',
+        AL: 'Albania',
+        AM: 'Armenia',
+        AO: 'Angola',
+        AQ: 'Antártida',
+        AR: 'Argentina',
+        AS: 'Samoa Americana',
+        AT: 'Austria',
+        AU: 'Australia',
+        AW: 'Aruba',
+        AX: 'Islas de Åland',
+        AZ: 'Azerbaiyán',
+        BA: 'Bosnia y Herzegovina',
+        BB: 'Barbados',
+        BD: 'Bangladesh',
+        BE: 'Bélgica',
+        BF: 'Burkina Faso',
+        BG: 'Bulgaria',
+        BH: 'Bahrein',
+        BI: 'Burundi',
+        BJ: 'Benin',
+        BL: 'San Bartolomé',
+        BM: 'Islas Bermudas',
+        BN: 'Brunéi',
+        BO: 'Bolivia',
+        BQ: 'Bonaire, San Eustaquio y Saba',
+        BR: 'Brazil',
+        BS: 'Bahamas',
+        BT: 'Bhután',
+        BW: 'Botsuana',
+        BY: 'Bielorrusia',
+        BZ: 'Belice',
+        CA: 'Canadá',
+        CC: 'Islas Cocos (Keeling)',
+        CD: 'República Democrática del Congo',
+        CF: 'República Centroafricana',
+        CG: 'República del Congo',
+        CH: 'Suiza',
+        CI: 'Costa de Marfil',
+        CK: 'Islas Cook',
+        CL: 'Chile',
+        CM: 'Camerún',
+        CN: 'China',
+        CO: 'Colombia',
+        CR: 'Costa Rica',
+        CU: 'Cuba',
+        CV: 'Cabo Verde',
+        CW: 'Curazao',
+        CX: 'Isla de Navidad',
+        CY: 'Chipre',
+        CZ: 'República Checa',
+        DE: 'Alemania',
+        DG: 'Diego García',
+        DJ: 'Yibuti',
+        DK: 'Dinamarca',
+        DM: 'Dominica',
+        DO: 'República Dominicana',
+        DZ: 'Argelia',
+        EA: 'Ceuta y Melilla',
+        EC: 'Ecuador',
+        EE: 'Estonia',
+        EG: 'Egipto',
+        EH: 'Sahara Occidental',
+        ER: 'Eritrea',
+        ES: 'España',
+        ET: 'EtiopíaEtiopía',
+        EZ: 'Eurozona',
+        FI: 'Finlandia',
+        FJ: 'Fiyi',
+        FK: 'Islas Malvinas',
+        FM: 'Micronesia',
+        FO: 'Islas Feroe',
+        FR: 'Francia',
+        GA: 'Gabón',
+        GB: 'Reino Unido de Gran Bretaña e Irlanda del Norte',
```

Comment:
I think we don't need the long name

```suggestion
        GB: 'Reino Unido',
```